common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/IdentifiersGenerators.scala

```diff
@@ -52,8 +54,22 @@ trait IdentifiersGenerators extends RandomStrings {
       identifierType = IdentifierType("isbn")
     )
 
+  private val MIRO_ID_PREFIXES: Seq[Char] = Seq(
```

Comment:
Why all the big letters?

Reply:
Because it's a constant and I've written too much C and it has rotted my brain - I'll change it to `MiroIdPrefixes` as per https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

Reply:
Fwiw I haven't seen `ConstantsLikeThis` once in our code base. I tend to just use `standardCamelCaseForConstants` (there are loads of examples, they are all over the place), and don't find it particularly confusing / misleading, but can understand if people not into it.

Reply:
I was just writing what @warrd said, if we are going to come up with style rules, we need to be able to lint them otherwise these conversations become often and boring

Reply:
OK sure - I certainly can't be bothered with linting since we have scalafmt. That said this conversation is evidence that we do have style rules, just undocumented ones that are different to the ones that someone who was looking for Scala style guidance would find, no? 😛

Reply:
I would probably agree with that statement, yep :see_no_evil:

Reply:
But I kind of think that is unavoidable for any team of devs, as no guidelines covers everything. It might be easier to follow the official scala style guide for these sort of cases, but starting at this point would probably lead to lots of inconsistencies (not that in this case I think it particularly matters if we were slightly inconsistent tho)